src/bfbs_gen_lua.cpp

```diff
@@ -153,8 +153,8 @@ class LuaBfbsGenerator : public BaseBfbsGenerator {
         code += "  if type(buf) == \"string\" then\n";
         code += "    buf = flatbuffers.binaryArray.New(buf)\n";
         code += "  end\n";
-        code += "\n";
-        code += "  local n = flatbuffers.N.UOffsetT:Unpack(buf, offset)\n";
+          code += "\n";
```

Comment:
indentation?

Reply:
That was to trigger the bot to see how it would report the issue :)